kubernetes/samples/scripts/domain-lifecycle/startServer.sh

```diff
@@ -0,0 +1,225 @@
+# !/bin/sh
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+source ${scriptDir}/helper.sh
+if [ "${debug}" == "true" ]; then set -x; fi;
+set -eu
+
+function usage() {
+
+  cat << EOF
+
+  This script starts a WebLogic managed server in a domain either by increasing
+  the value of 'spec.clusters[<cluster-name>].replicas' by '1' or by updating the
+  'spec.managedServers[<server-name>].serverStartPolicy' attribute of the domain
+  resource or both as needed. The 'spec.clusters[<cluster-name>].replicas' value can
+  be kept constant by using '-k' option.
```

Comment:
If you choose to update README.md to have significant additional details,  add a sentence like 'See README.md for additional details.'